docs/standard/native-interop/abi-support.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Native interoperability ABI support - .NET
+description: Defines the current support for interoperability with various ABIs.
+ms.date: 10/27/2024
+---
+# Native interoperability ABI support
+
+The [Application Binary Interface](https://wikipedia.org/wiki/Application_binary_interface) (ABI) is the interface which run-times and operating systems express low-level binary details. This can include calling conventions (that is, how parameters are passed and results returned), exception handling, as well as symbol mangling. The below list contains the names of languages, run-times and even general technologies since these names tend to represent what users will be working with when looking for guidance on interoperability.
+
+## C
+
+The C language represents a stable ABI across all platforms where .NET is supported. It is what most .NET interop APIs assume is the target. This is the recommended target language for most interop scenarios using .NET.
+
+In .NET 7+, <xref:System.Runtime.InteropServices.LibraryImportAttribute> provides source generated support for calling C functions. When targeting .NET 6 or earlier, use <xref:System.Runtime.InteropServices.DllImportAttribute>. Refer to [Interop best practices](./best-practices.md) for additional guidance.
+
+Many virtual machine based languages define a foreign function interface (FFI) in C to interoperate with other platforms. A list of examples is below.
+
+* Java Virtual Machine (JVM)
+* CPython
+
+Additional links:
+
+* [CsWin32](https://github.com/microsoft/CsWin32) a source generator for access Windows Win32 API
+* [Java binding generator](https://github.com/dotnet/java-interop)
+
```

Comment:
```suggestion
Additional links:

* [CsWin32](https://github.com/microsoft/CsWin32) a source generator for access Windows Win32 API

## Java

Java Virtual Machine (JVM) defines a foreign function interface (FFI) in C to interoperate with other platforms. Interoperability between .NET and Java can be achieved via this interface.

Additional links:

* [Java binding generator](https://github.com/dotnet/java-interop)
```
I think Java (and potentially Python) should have its own paragraphs.

Reply:
Okay. Added Java and Python sections.